packages/dispatcher-converter/src/util/FileOperations.js

```diff
@@ -1887,11 +1896,9 @@ class FileOperations {
      * @param file
      */
     getFileContentsArray(file, recursive = true) {
-        let fileContents = "";
         let fileContentsArray = "";
 
-        fileContents = this.getContentFromFile(file, recursive);
-        fileContentsArray = fileContents.split(os.EOL);
+        fileContentsArray = util.getXMLContentSync(file, recursive);
```

Comment:
The method [getXMLContentSync](https://github.com/adobe/aem-cloud-service-source-migration/blob/master/packages/commons/src/util.js#L77) only takes in a file path, the `recursive` param should be removed.

Reply:
Done